docs/CONTRIBUTING.md

```diff
@@ -141,6 +141,13 @@ Example:
     GOOD: * [A Way-backed Interesting Book](https://web.archive.org/web/20211016123456/http://example.com/) - John Doe (HTML) *(:card_file_box: archived)*
     ```
 
+### Alphabetical order
+
+- When there are multiple titles beginning with the same letter order them by the second, and so on. For example: `aa` comes before `ab`.
+- `one two` comes before `onetwo`
+
+If you see a misplaced link, please reorder it and submit a PR.
```

Comment:
The linter won't allow a misplaced entry, maybe replace this with something about checking the linter?

Reply:
```suggestion
If you see an misplaced link, check the linter error message to know which lines should be swapped.
```

Can this be okay?

Reply:
except for "an" it's fine